.travis.yml

```diff
@@ -6,32 +6,29 @@ addons:
 matrix:
   include:
     - node_js: "14"
-      env: OPENPGP_NODE_JS='14' OPENPGPJSTEST='lint'
+      script: npm run lint
     - node_js: "14"
-      env: OPENPGP_NODE_JS='14' OPENPGPJSTEST='test-type-definitions'
+      script: npm run test-type-definitions
     - node_js: "10"
-      env: OPENPGP_NODE_JS='10' OPENPGPJSTEST='unit'
+      script: npm test
     - node_js: "12"
-      env: OPENPGP_NODE_JS='12' OPENPGPJSTEST='unit'
+      script: npm test
     - node_js: "14"
-      env: OPENPGP_NODE_JS='14' OPENPGPJSTEST='unit'
+      script: npm test
     - node_js: "14"
-      env:  OPENPGP_NODE_JS='14' OPENPGPJSTEST='unit' LIGHTWEIGHT=1
+      script: npm test -- --grep lightweight
     - node_js: "14"
-      env: BROWSER='"firefox_68"' OPENPGPJSTEST='browserstack'
+      env: BROWSER='{"browser":"Firefox", "browser_version":"68.0", "os":"OS X", "os_version":"High Sierra"}'
     - node_js: "14"
-      env:  BROWSER='"chrome_68"' OPENPGPJSTEST='browserstack' LIGHTWEIGHT=1
+      env: BROWSER='{"browser":"Chrome", "browser_version":"68.0", "os":"OS X", "os_version":"High Sierra"}' LIGHTWEIGHT=1
     - node_js: "14"
-      env: BROWSER='"chrome_68"' OPENPGPJSTEST='browserstack'
+      env: BROWSER='{"browser":"Chrome", "browser_version":"68.0", "os":"OS X", "os_version":"High Sierra"}'
     - node_js: "14"
-      env: BROWSER='"safari_11_1"' OPENPGPJSTEST='browserstack'
+      env: BROWSER='{"browser":"Safari", "browser_version":"11.1", "os":"OS X", "os_version":"High Sierra"}'
```

Comment:
Can we add targets for recent versions of the browsers? Latest Chrome is v88, and v14 for Safari

Reply:
Yeah; this PR is meant to be functionally equivalent, but we can bump the versions in a later PR :+1: